catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoEntry.java

```diff
@@ -37,6 +39,7 @@ private GeoEntry(final Builder builder) {
     population = builder.population;
     alternateNames = builder.alternateNames;
     countryCode = builder.countryCode;
+    importLocation = builder.importLocation;
```

Comment:
✏️  Directly accessing private field instead of going through getter

Reply:
I followed the pattern in the existing class.